src/components/Staking/StakingComparison.tsx

```diff
@@ -63,12 +64,24 @@ const StyledPoolGlyph = styled(PoolGlyph)`
     fill: ${({ theme }) => theme.colors.stakingBlue};
   }
 `
-
-const StakingComparison = ({ page, className }) => {
+export interface IProps {
```

Comment:
Lets reorganize it a bit to improve the readability and define a little better some types. We can put this on top of the function definition:
 
```ts
interface DataType {
  title: TranslationKey
  linkText: TranslationKey
  to: string
  matomo: EventOptions
  color: string
  glyph: ReactNode
}

type StakingPageType = "solo" | "saas" | "pools"

interface StakingPageData extends DataType {
  content: TranslationKey
}

export interface IProps {
  page: StakingPageType
  className?: string
}
```

and then in `data` you can just do:
```ts
const data: {
    [key in StakingPageType]: Array<StakingPageData>
  } = {
   ...
```

Reply:
I was doubting here, since the pages are hardcoded in multiple places anyway. But I agree. Thanks